wagtail/admin/views/account.py

```diff
@@ -138,10 +138,18 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
+    # title = gettext_lazy("Name and Email")
     order = 100
     form_class = NameEmailForm
 
+    @property
+    def title(self):
+        from wagtail.admin.views.account import email_management_enabled
+
+        if email_management_enabled():
+            return gettext_lazy("Name and Email")
+        return gettext_lazy("Name")
```

Comment:
We might not need lazy anymore - as this is a function.

```suggestion
        return _("Name")
```